tests/tools/betting_strategies/test_kelly.py

```
@@ -420,6 +445,9 @@ def test_compare_kellys_full(estimated_p_yes: Probability, confidence: float) ->
             allow_multiple_bets=False,
             allow_shorting=False,
             multicategorical=False,
+            get_buy_token_amount=lambda bet_amount, outcome_index: calculate_buy_outcome_token(
+                bet_amount, outcome_index, pool_sizes, market.fees
```

Ruff (0.12.2)

Function definition does not bind loop variable `pool_sizes`

(B023)